__tests__/unit/cloudstore.android.test.js

```diff
@@ -0,0 +1,99 @@
+import * as CloudStore from '../../src/cloudstore';
+import * as GDriveCloudStorage from '../../src/GDriveCloudStorage';
```

Comment:
Why not just unit test your new `GDriveCloudStorage` module instead? There should be no big difference in the `CloudStore` logic if the `GDriveCloudStorage` api is compatible with the `AsyncStorage` api. That way you also don't have to worry about platform detection.

Reply:
Hmm yeh I'm not sure there is much value in that as I'd just be mocking the functions from `react-native-google-drive-api-wrapper` and `@react-native-community/google-signin`. The tests I wrote were passing so I pretty sure it's compatible with `AsyncStorage`. Maybe just keep but skip them for now as if they pass it's not really a guarantee it works in an application yet

Reply:
Mmmh yeah. Agreed there is little value in testing `GDriveCloudStorage` since it has very little logic. Have you tested this on the android device yet?